### ISSUE TITLE
Icon is not displayed

### ISSUE THREAD

**Issue by h-brinkmann** [enovation/moodle-atto_teamsmeeting#22]
Hello,

I have installed the newest plugin version 1.21 from a zip file into a Moodle 3.82 instance.
The installation seemed to be correct. The files are in the recommended path and in Moodle the plugin is installed and can be configured. No error messages occured. Then i added the Icon accorded to the Documentation.

The Problem is, the icon is not displayed in the editor. 
Then i deleted the plugin and reinstalled it twice, but the result was always the same. The icon never showed up.

Any idea what went wrong?

**Comment by alfredonodo** [comment]
Hi,
I have the same issue with latest version of moodle 3.9.2 and plugin 1.2.1. The teams icon is not displayed into atto editor.

**Comment by pachainti** [comment]
I have the same problem with moodle 3.9.2 and team plugin 1.2.1. Please fix it as soon as possible.
Thank you

**Comment by weilai-irl** [comment]
Hi @h-brinkmann, @alfredonodo, @pachainti,

I checked the code and I don't see anything obviously wrong within the code. The issue is most likely in configuration missing, i.e. the plugin has not been configured to be included in the atto editor toolbar. To do it:
1. Login to Moodle site using site admin account.
2. In the "Site administration" section, go to "Plugins" -> "Text editors" -> "Atto HTML editor".
3. Find "Toolbar config" setting on the page. It defines what icons are shown on the atto editor toolbar, one section per line.
4. You will need to add value "teamsmeeting" to one of the sections, e.g. "links", so that the "links" line looks like "links = link, teamsmeeting".
5. Save changes.

Then if you go to any page containing an atto editor, the Teams icon should show up.

Please confirm if this fixes the issue.

Regards,
Lai

**Comment by alfredonodo** [comment]
@weilai-irl you are right, now it is visible. I'm sorry, but I did not find such information before to ask.

**Comment by pachainti** [comment]
Hi,
I can confirm this too. Is there any documentation about it?
Thank you

**Comment by weilai-irl** [comment]
@alfredonodo, @pachainti, thank you for your confirmation.

@pachainti, this is standard behaviour of atto editor plugins in Moodle, which is described at https://docs.moodle.org/39/en/Text_editor#Atto_editor_toolbar_layout. It's not a feature introduced by the plugin itself, so was not described in the documentation of the plugin

Regards,
Lai

**Comment by pachainti** [comment]
@weilai-irl you are right, however it could be useful if you add more information (that link) on the plugin page.
I tried the plugin, it works well. However, in my opinion it would be good to have a plugin that allow to add an activity or a resource as those of [BBB](https://moodle.org/plugins/mod_bigbluebuttonbn) and [jitsi](https://moodle.org/plugins/mod_jitsi).

**Comment by h-brinkmann** [comment]
Hi, @weilai-irl,

As i wrote in my first post i added the icon accorded to the documentation.

That means exactly that way you propose it. I add the value "teamsmeeting" in the atto-toolbar-config-settings as described (links = link, teamsmeeting), but that makes no difference. The icon wasn´t displayed. 

In the plugIn overview _editorsettingsatto_ section you can see the icon. But not in the atto-editor itself. 

Regards
Heinz

**Comment by weilai-irl** [comment]
Hi @h-brinkmann,

I can't think of anything other reason why it's not working in the Atto HTML editor itself.

The only other thing that I can think of is Atto HTML editor is not actually used. Could you make sure "Atto HTML editor" is selected as the default editor please. To do this, go to https://URL.TO.MOODLE/admin/settings.php?section=manageeditors, and ensure "Atto HTML editor" is enabled, and placed before other editors.

Regards,
Lai

**Comment by weilai-irl** [comment]
Hi @h-brinkmann,

Could you attach some screenshots showing the error please. Please include at least two screenshots:
 1. Atto editor configuration to shows the atto-toolbar-config-settings settings. 
 2. A page containing the atto editor and its complete set of buttons, on a Moodle page, e.g. course settings page.

Thank you.

Regards,
Lai

**Comment by weilai-irl** [comment]
Hi @pachainti, 

Thank you for your suggestion.

I agree that in longer term, it may be more desirable to have this implemented as a full feature activity plugin with support of all Moodle activity module features, however, it will effectively increase the scope of the project by a few times (for grade book integration, completion integration, backup and restore etc.). This isn't the original purpose of this plugin, which was to provide a lightweight solution to allow users to create Microsoft Teams meetings from anywhere in Moodle. Besides, from what I can see, we currently don't have the resource to facilitate it.

Please log this as a separate feature request to get the ball started rolling. If the feature request gets many votes, it may become reality sometime soon.

Regards,
Lai
